test/preparser.js

```diff
@@ -37,7 +38,63 @@ describe('preparser', function () {
       }
     });
 
-    it.skip('should convert the same variable twice', function () {
+    it('concatenate variables with no space', function () {
```

Comment:
Really nit-picky, but could you please add `should` to the start of each string? I reads better on `npm test`.